shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -322,6 +319,12 @@ private void popSystemNavigator() {
     }
   }
 
+  private boolean clipboardHasText() {
+    ClipboardManager clipboard =
+        (ClipboardManager) activity.getSystemService(Context.CLIPBOARD_SERVICE);
+    return clipboard.hasText();
```

Comment:
I think is now deprecated:
https://developer.android.com/reference/android/content/ClipboardManager#hasText()

Wondering if we should replace it with `hasPrimaryClip()`:
https://developer.android.com/reference/android/content/ClipboardManager#hasPrimaryClip()

Reply:
See https://github.com/flutter/engine/pull/24021#discussion_r574131148 since @longcatislooong mentioned the same thing.